api/types/network/port.go

```diff
@@ -10,24 +10,24 @@ import (
 	"unique"
 )
 
-// NetworkProtocol represents a network protocol for a port.
-type NetworkProtocol string
+// IPProtocol represents a network protocol for a port.
+type IPProtocol string
 
 const (
-	TCP  NetworkProtocol = "tcp"
-	UDP  NetworkProtocol = "udp"
-	SCTP NetworkProtocol = "sctp"
+	TCP  IPProtocol = "tcp"
+	UDP  IPProtocol = "udp"
+	SCTP IPProtocol = "sctp"
```

Comment:
As a follow-up, perhaps we should add a comment to this block to;

- Outline these are the protocols currently supported by the daemon
- (?) But that the list may be non-exhaustive (at least, I recall some discussion about the daemon being the source of truth for this, not the API)